packages/downloader/download.py

```
@@ -0,0 +1,126 @@
+            try:
+                with open(progress_file, 'r') as ro:
+                    progress_data = json.load(ro)
+                progress_data["status"] = "finished"
+                
+                # Write atomically
+                temp_path = OUTPUTS_DIR / f'{info_dict["id"]}.json.tmp'
+                with open(temp_path, 'w') as wo:
+                    json.dump(progress_data, wo)
+                os.rename(temp_path, progress_file)
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)